zebra/zebra_fpm_netlink.c

```diff
@@ -274,21 +274,8 @@ static int netlink_route_info_fill(struct netlink_route_info *ri, int cmd,
 				   rib_dest_t *dest, struct route_entry *re)
 {
 	struct nexthop *nexthop;
-	struct rib_table_info *table_info =
-		rib_table_info(rib_dest_table(dest));
-	struct zebra_vrf *zvrf = table_info->zvrf;
-
-	memset(ri, 0, sizeof(*ri));
```

Comment:
was DELROUTE processing depending on this work's being done before the 'return' just below?